documentation/book/assembly-kafka-mirror-maker.adoc

```diff
@@ -0,0 +1,23 @@
+// This assembly is included in the following assemblies:
+//
+// getting-started.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='kafka-mirror-maker-{context}']
+= Kafka Mirror Maker
+
+The Cluster Operator deploys one or more Kafka Mirror Maker instances, which can be used to replicate data between Kafka clusters.
```

Comment:
I think instances is a bit confusing here. Are we talking about replicas? Or deployments? I think we need to clarify it, since in the chapter about replicas you talked about instances as well.